RecoDataProducts/inc/CaloHit.hh

```diff
@@ -37,9 +42,9 @@ namespace mu2e {
           int    crystalId_;
           int    nSiPMs_;
           float  time_;             
-          float  timeErr_;             
           float  eDep_;        
           float  eDepErr_;        
+          float  timeErr_;             
```

Comment:
Is this change just for aesthetic reasons?

I think that root's automatic schema evolution will manage this correctly but it may mean that reading old files with new code will be slower.  Is that an issue that we need to think about?  If that's only a small transient problem then this is a good time for the change.

Reply:
I don't think we will have an issue with older datasets as there were other breaking changes, no? I changed that because in the online reco we don't fill the *Err_ parameters and @bechenard suggested me to make this change to optimize the memory allocation.

Reply:
Yes, we are making other breaking changes.  That's why I said this is a good time for a change.   I will mark this resolved but I have a  follow up offline.